client/src/pages/UserAvailability.js

```diff
@@ -0,0 +1,165 @@
+            value:"Thursdays",
+            state:false,
+            color:"rgb(0 0 0 / 15%)"
+        },
+
+        {
+            value:"Fridays",
+            state:false,
+            color:"rgb(0 0 0 / 15%)"
+        },
+
+        {
+            value:"Saturdays",
+            state:false,
+            color:"rgb(0 0 0 / 15%)"
+        }
+    ])
+
+    const toggleCheckbox = (event) => {
+        const temp = days.slice()
+        const idx = event.target.value
+        const prevState = temp[idx].state 
+        temp[idx].state = !prevState
+        temp[idx].color = temp[idx].state ? "#000000" : "rgb(0 0 0 / 15%)"
+
+        setDays(temp)
+    }
+
+    return (
+        <FormGroup row>
+            {days.map((date,index)=>(
+                <FormControlLabel
+                key={index}
+                value={index}
+                checked= {date.state}
+                control={<Checkbox  color="primary"/>}
+                label={<Typography variant="subtitle2" className={classes.checkBoxText}>{date.value}</Typography>}
+                labelPlacement="bottom"
+                onChange={toggleCheckbox}
+                classes={{root:classes.checkBox}}
+                style={{color:date.color}}
+                />
+            ))}
+        </FormGroup>
+    )
+}
+
+function UserAvailability() {
+
+    const classes = useStyles();
+
+    const path = "/home/event"
+    
+    const [startTime,setStartTime] = React.useState(9)
+
+    const [endTime,setEndTime] = React.useState(17)
+
+    const handleStartTimeChange = (event) => {
+        setStartTime(event.target.value)
+    }
+
+    const handleEndTimeChange = (event) => {
+        setEndTime(event.target.value)
+    }
+
+    return (
+        <Onboarding title="Set your availability" activeStep={3} path={path}>
+            <Grid container item xs={12} alignItems="center">
+                <Grid item xs={12} style={{marginBottom:10}}>
+                    <Typography variant="subtitle2">Available Hours:</Typography>
+                </Grid>
+                <Grid container item xs={12} alignItems="center" spacing={3}>
+                    <Grid item xs={3}> 
+                        <DropdownSelect defaultValue={startTime} handler={handleStartTimeChange} options={timeOptions}/>
+                    </Grid>
+                    
+                    <Grid item><Typography variant="h6">-</Typography></Grid>
+
+                    <Grid item xs={3}>    
+                        <DropdownSelect defaultValue={endTime} handler={handleEndTimeChange}  options={timeOptions}/>
+                    </Grid>
+                </Grid>
+            </Grid>
+
+            <Grid container item xs={12} alignItems="center" className={classes.checkBoxContainer}>
+                <Grid item xs={12} style={{marginBottom:10}}>
+                    <Typography variant="subtitle2">Available Days:</Typography>
+                </Grid>
+                <Grid item container justify={"flex-start"} xs={12}>
```

Comment:
The attribute `justify` works in the same way that the `alignItems` attribute works in the above grid items/containers:`justify="flex-start"` instead of `justify={"flex-start"}` 🙂